src/envoy/http/authn/filter_context_test.cc

```diff
@@ -37,7 +37,7 @@ class FilterContextTest : public testing::Test {
 
   // This test suit does not use headers nor connection, so ok to use null for
   // them.
-  StrictMock<FilterContext> filter_context_{nullptr, nullptr};
+  StrictMock<FilterContext> filter_context_{nullptr, nullptr, nullptr};
```

Comment:
If change to const ref for config, you can pass the FilterConfig::default_instance instead of nullptr

Reply:
Done.